support/SubmitScripts/OceanClang.sh

```diff
@@ -103,5 +103,34 @@ spectre_load_modules
 
 chmod u+x ${SPECTRE_RUN_DIR}/runscript.${SLURM_JOBID}
 
-charmrun ++runscript ${SPECTRE_RUN_DIR}/runscript.${SLURM_JOBID} \
-         ${SPECTRE_COMMAND} --input-file ${SPECTRE_INPUT_FILE}
+checkpoints=0
+current_checkpoint=000000
+if [[ $checkpoints == 0 ]]; then
+  charmrun ++runscript ${SPECTRE_RUN_DIR}/runscript.${SLURM_JOBID} \
+          ${SPECTRE_COMMAND} --input-file ${SPECTRE_INPUT_FILE}
+  sleep 10s
+  # If a checkpoint is found add one to checkpoint and sumbit next job
+  if test -e "${PWD}/SpectreCheckpoint$current_checkpoint"; then
+    cp ../OceanClang.sh .
+    sed -i 's/checkpoints=0/checkpoints=1/' OceanClang.sh
+    sbatch OceanClang.sh
+  fi
+# Section to start from checkpoint
+elif [[ $checkpoints -gt 0 && $checkpoints -lt 1000000 ]]; then
+  ln -s ${PWD}/../SpectreCheckpoint$current_checkpoint .
+  charmrun ${SPECTRE_COMMAND} +restart SpectreCheckpoint$current_checkpoint \
+          --input-file ${SPECTRE_INPUT_FILE}
+  sleep 10s
+  # If the next checkpoint was created modify variables and submit next job
+  printf -v next_checkpoint %06d $checkpoints
+  if test -e "${PWD}/SpectreCheckpoint$next_checkpoint"; then
+    cp ../OceanClang.sh .
+    next_num_of_checkpoints=$(($checkpoints + 1))
+    #Updating variables for the next possible checkpoint
+    sed -i 's/checkpoints='$checkpoints'/'\
+'checkpoints='$next_num_of_checkpoints'/' OceanClang.sh
+    sed -i 's/current_checkpoint='$current_checkpoint'/'\
+'current_checkpoint='$next_checkpoint'/' OceanClang.sh
```

Comment:
For these 2 sed commands, indenting them breaks functionality

Reply:
That's fine.